dspace-iiif/src/main/java/org/dspace/app/iiif/service/WordHighlightSolrSearch.java

```diff
@@ -251,7 +248,7 @@ private String getCanvasId(JsonNode pagesNode) {
                 if (pageId != null) {
                     String[] identArr = pageId.asText().split("\\.");
                     // the canvas id.
-                    return "c" + identArr[1];
+                    return identArr[1];
```

Comment:
Solr indexing is out of scope here, but I just want to note that since the `pageId` returned in the [Solr Word Higlighting response](https://dbmdz.github.io/solr-ocrhighlighting/0.8.3/) may simply become a bitstream UUID in the future, we probably won't need to split on `\\.` to obtain a position value. There's really no way to address this here without reviewing search functionality and that's probably best done in a separate PR